ipaserver/install/kra.py

```diff
@@ -69,9 +68,19 @@ def install_check(api, replica_config, options):
 
 
 def install(api, replica_config, options):
-    if replica_config is None:
-        if not options.setup_kra:
-            return
+
+    if replica_config is not None and not replica_config.setup_kra:
+        return
+
+    install_kra_replica = False
+    kra = krainstance.KRAInstance(api.env.realm)
+
+    if kra.is_installed() and replica_config is not None:
+        install_kra_replica = False
+    elif not kra.is_installed():
+        install_kra_replica = True
+
```

Comment:
I am not sure what difference checking replica_config would make at the moment.
It was used previously because we did not allow to install a new KRA from scratch on a replica, but now we are so it is not really relevant to check I think.